Add Connectors and ML updates for 2.9 (#4554)
opensearch-project/documentation-website: _ml-commons-plugin/ml-dashboard.md

  - /ml-commons-plugin/ml-dashbaord/
---

> The ML dashboard was taken out of experimental status and released to General Availability in OpenSearch 2.9.  

Reviewer: Do we want to include this note or do we just want to omit the experimental note? If we're including this, I think a note would be more appropriate than a warning.

Author: I'm okay changing it to a note.

> The ML dashboard was taken out of experimental status and released to General Availability in OpenSearch 2.9.  

Reviewer 2: ```suggestion
The ML dashboard was taken out of experimental status and released as Generally Available in OpenSearch 2.9.  
```

> {: .warning}

Author: ```suggestion
{: .note}
```

## Next steps

> For more information about how to manage ML models in OpenSearch, see [ML framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework/).

Reviewer 2: ```suggestion
For more information about how to manage ML models in OpenSearch, see [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-serving-framework/).
```
